projects/NETCore.md

```diff
@@ -1,14 +1,14 @@
 # .NET Core
 
-.NET Core is the small optimized runtime that is the basis of [ASP.NET Core 5](http://www.dotnetfoundation.org/aspnet-5). It currently runs on Windows, and will be extended to support Linux and Mac. It is a high-performance and modular design, and supports full side by side to make it easy to adopt new .NET Core versions without affecting other apps. These products are actively developed by the .NET team and in collaboration with a community of open source developers. Together we are dedicated to improving and extending the .NET platform with new features and for new scenarios.
+.NET Core is the small optimized runtime that is the basis of [ASP.NET Core](http://www.dotnetfoundation.org/aspnet-core). It currently runs on Linux, Mac, and Windows. It is a high-performance and modular design, and supports full side by side to make it easy to adopt new .NET Core versions without affecting other apps. These products are actively developed by the .NET team and in collaboration with a community of open source developers. Together we are dedicated to improving and extending the .NET platform with new features and for new scenarios.
 
 .NET Core has two major components. It includes a small runtime that is built from the same codebase as the .NET Framework CLR. The .NET Core runtime includes the same GC and JIT (RyuJIT), but doesn’t include features like Application Domains or Code Access Security. The runtime is delivered on NuGet, via the Microsoft.CoreCLR package.
 
 .NET Core also includes the base class libraries. These libraries are largely the same code as the .NET Framework class libraries, but have been factored (removal of dependencies) to enable us to ship a smaller set of libraries. These libraries are shipped as System.* NuGet packages on NuGet.org.
 
 ## Project Details
 
-* [Project Info Site](https://dotnet.github.io/core)
+* [Project Info Site](https://dotnet.github.io)
 * Project Code Repos:  [.NET Core Framework](https://github.com/dotnet/corefx), [.NET Core Common Language Runtime]( https://github.com/dotnet/coreCLR)
```

Comment:
I can add the first if you'd like. 

Should the doc link go to the 'core' or the .NET home page? 

Also, @BethMassi can you review the text at line 5? I want to make sure that is still accurate in terms of how it positions core in contrast to the full framework.